libethereum/Executive.cpp

```diff
@@ -81,13 +88,17 @@ void StandardTrace::operator()(uint64_t _steps, Instruction inst, bigint newMemS
 		assert(m_lastInst.size() == ext.depth);
 		m_lastInst.push_back(inst);
 		newContext = true;
+		m_levels.append(std::to_string(_steps));
+		r["levels"] = m_levels;
```

Comment:
m_levels is an array which contain vmtrace index that correspond to depth change. This is used to build the callstack